### PR TITLE
[SD-2974] Publish subsequent takes to subscribers that have received previously.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: server/apps/publish/archive_publish.py [hunk 1]
```diff
@@ -653,11 +655,17 @@ def get_subscribers(self, doc, target_media_type):
...
-        subscribers, subscribers_yet_to_receive = [], []
+        subscribers, subscribers_yet_to_receive, takes_subscribers = [], [], []
         first_take = None
         # Step 1
         subscribers = list(get_resource_service('subscribers').get(req=None, lookup={'is_active': True}))
 
+        if doc.get(ITEM_TYPE) in [CONTENT_TYPE.COMPOSITE] and doc.get(PACKAGE_TYPE) == TAKES_PACKAGE:
+            # Step 1a
+            query = {'$and': [{'item_id': doc[config.ID_FIELD]},
+                              {'publishing_action': {'$in': ['published', 'corrected']}}]}
```
Comment (sivakuna-aap) [inline server/apps/publish/archive_publish.py]: Constants for the Item State

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**